databricks_cli/dbfs/cli.py

```diff
@@ -140,7 +120,14 @@ def copy_to_dbfs_recursive(src, dbfs_path_dst, overwrite):
         if os.path.isdir(cur_src):
             copy_to_dbfs_recursive(cur_src, cur_dbfs_dst, overwrite)
         elif os.path.isfile(cur_src):
-            copy_to_dbfs(cur_src, cur_dbfs_dst, overwrite)
+            try:
+                put_file(cur_src, cur_dbfs_dst, overwrite)
+                click.echo('{} -> {}'.format(cur_src, cur_dbfs_dst))
+            except HTTPError as e:
+                if e.response.json()['error_code'] == DbfsErrorCodes.RESOURCE_ALREADY_EXISTS:
+                    click.echo(e.response.json())
```

Comment:
Shall we make the error message clearer? Right now it is

~~~
{u'message': u'A file or directory already exists at the input path dbfs:/home/meng/path.', u'error_code': u'RESOURCE_ALREADY_EXISTS'}
~~~

A more consistent message would be

~~~
dbfs:/home/meng/tmp/dbfsBytesRead/path already exists. Skip.
~~~